### ISSUE TITLE
Change log for Focus Visible SC

### ISSUE THREAD

**Issue by giacomo-petri** [w3c/wcag#3468]
On November 10, 2019, Focus Visible SC was initially promoted to Level A, as indicated in the change log. However, on April 7, 2023, it was subsequently downgraded to Level AA (source: https://github.com/w3c/wcag/commits/main/understanding/20/focus-visible.html), without any change log update.

Since my initial check of the change log did not yield the information I was seeking, I took a closer look at the commit history to pinpoint when the SC's level was adjusted.

I recommend updating the change log to reflect this change by either removing the November 10, 2019 entry or adding a new entry that documents the reversal of the level to its original state.

**Comment by yatil** [comment]
→ Link to changelog: https://www.w3.org/TR/WCAG22/#changelog

Yes, this changelog feels very hit-and-miss and also fails to mention the removal of 4.1.1 Parsing.

**Comment by venkatvemuri28** [comment]
I have raised a pull request here 
https://github.com/w3c/wcag/pull/4668

**Comment by mbgower** [comment]
I made this response to the PR. Should have put it here:

> We do not normally record contents that change before the FPWD or between the FPWD and the publishing of the Rec that are not incorporated as part of the published rec (my emphasis in bold):
> 
> > substantive changes incorporated into WCAG 2.2 since WCAG 2.1, as well as changes made to 2.2 since its original publication on 05 October 2023
> 
> Since the change in level of Focus Visible was reverted before publication date, I think neither entry is necessary.
> 
> As an example, the proposed Focus Appearance (Minimum) that was originally brought in as level AA is not recorded as moving to a AAA; we only record the addition and the name change. There is no mention of Focus Appearance (Enhanced) which also appeared in the August 2020 draft but was later removed.

I agree that 4.1.1 should be in the change log. I suggest we put in a date of 4.1.1 Parsing being removed on Jan 25, 2023.

**Comment by alastc** [comment]
For the original issue, I don't think we'd put in two entries which cancel out.

We originally put that into the changelog when it was going to be put at A, but as it was reverted, the changelog wouldn't be representing the changes from 2.1 to 2.2. I think the original entry was intentionally removed.

However, yatil has a point on parsing, we should include that. @venkatvemuri28 would you mind changing that? There must be a PR in the history somewhere?

**Comment by kfranqueiro** [comment]
For the record, the original issue reported here RE Focus Visible was resolved by https://github.com/w3c/wcag/pull/4123, where the changelog item about promoting to Level A was removed, rather than adding another item to document the reversal.

**Comment by venkatvemuri28** [comment]
Thanks for clarifying. Just to confirm — after removing the previous Focus Visible entries, should I add the following line?

>   <li>2023-01-25: 4.1.1 Parsing marked as obsolete (removed in Candidate Recommendation Draft).</li>
or

>   <li>2023-01-25: 4.1.1 Parsing marked as obsolete and removed in the WCAG 2.2 Candidate Recommendation Draft.</li>


Please let me know if you’d prefer a different wording or format.

**Comment by alastc** [comment]
I think the second option, just "2023-01-25: 4.1.1 Parsing marked as obsolete and removed."
